lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1802,6 +1810,13 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => "Controls auto-instrumentation of the Thread class at start up to automatically add tracing to all Threads created in the application."
         },
+        :'thread_ids_enabled' => {
+          :default => false,
+          :public => false,
+          :type => Boolean,
+          :allowed_from_server => false,
+          :description => "If enabled, will append the current Thread and Fiber object ids onto the segment names of segments created in Threads and concurrent-ruby"
+        },
```

Comment:
It's set to `public => false` because @tannalynn and I were thinking it would only be used when an agent developer was trying to debug an issue. Do we think it would have a more public appeal? Should it be exposed to the public?

Reply:
Thank you both for clarifying! I like leaving it private for now. We can revisit if/when we have a public Fiber instrumentation toggle in future.